hazelcast-hibernate5/src/test/java/com/hazelcast/hibernate/region/HazelcastQueryResultsRegionTest.java

```diff
@@ -1,136 +0,0 @@
-import com.hazelcast.core.HazelcastInstance;
-import com.hazelcast.hibernate.local.LocalRegionCache;
-import com.hazelcast.hibernate.local.LocalRegionCacheTest;
-import com.hazelcast.test.HazelcastSerialClassRunner;
-import com.hazelcast.test.annotation.ParallelJVMTest;
-import com.hazelcast.test.annotation.QuickTest;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
-import org.mockito.invocation.InvocationOnMock;
-import org.mockito.stubbing.Answer;
-
-import java.util.Properties;
-import java.util.concurrent.TimeUnit;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-import static org.mockito.Mockito.anyString;
-import static org.mockito.Mockito.atLeastOnce;
-import static org.mockito.Mockito.eq;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-
-@RunWith(HazelcastSerialClassRunner.class)
-@Category({QuickTest.class, ParallelJVMTest.class})
-public class HazelcastQueryResultsRegionTest {
-
-    private static final String REGION_NAME = "query.test";
-
-    private int maxSize = 50;
-    private int timeout = 60;
-
-    private MapConfig mapConfig;
-    private Config config;
-
-    private HazelcastInstance instance;
-    private HazelcastQueryResultsRegion region;
-
-    @Before
-    public void setUp() throws Exception {
-        mapConfig = mock(MapConfig.class);
-        EvictionConfig evictionConfig = new EvictionConfig();
-        evictionConfig.setMaxSizePolicy(MaxSizePolicy.PER_NODE);
-        evictionConfig.setSize(maxSize);
-        when(mapConfig.getEvictionConfig()).thenReturn(evictionConfig);
-
-        when(mapConfig.getTimeToLiveSeconds()).thenReturn(timeout);
-
-        config = mock(Config.class);
-        when(config.findMapConfig(eq(REGION_NAME))).thenReturn(mapConfig);
-
-        Cluster cluster = mock(Cluster.class);
-        when(cluster.getClusterTime()).thenAnswer(new Answer<Long>() {
-            @Override
-            public Long answer(InvocationOnMock invocation) throws Throwable {
-                return System.currentTimeMillis();
-            }
-        });
-
-        instance = mock(HazelcastInstance.class);
-        when(instance.getConfig()).thenReturn(config);
-        when(instance.getCluster()).thenReturn(cluster);
-
-        region = new HazelcastQueryResultsRegion(instance, REGION_NAME, new Properties());
-    }
-
-    /**
-     * Verifies that the region retrieved the MapConfig and set its timeout from the TTL.
-     * Also verifies that the nested LocalRegionCache retrieved the configuration,
-     * but did not register a listener on any ITopic.
-     */
-    @Test
-    public void testCacheHonorsConfiguration() {
```

Comment:
Is that behavior tested else where over cache behavior (max size, ttl, etc.)? Or else why we remove this?

Reply:
Yes, we test if cache works properly at a higher level, therefore no need for such fragile testing